packages/audio-block/src/AudioBlock.spec.ct.tsx

```diff
@@ -219,4 +221,14 @@ describe('Audio Block', () => {
         mount(<AudioBlockWithStubs />);
         cy.get(ViewModeAddonsSelector).should('not.exist');
     });
+
+    it.only('should add the file name to the title when the title is empty', () => {
```

Comment:
```suggestion
    it('should add the file name to the title when the title is empty', () => {
```

Reply:
Is this actually expected behaviour or should it always overwrite when the asset changes? Would be good to know if this has already been discussed with Martin/Schirmi

Reply:
I has been discussed last year and it should just be set when the asset gets added and no title is set. But yeah will add a second test for that